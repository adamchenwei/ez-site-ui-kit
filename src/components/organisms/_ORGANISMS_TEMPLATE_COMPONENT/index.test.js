import React from 'react';
import renderer from 'react-test-renderer';
import OrganismsTemplateComponent from './index';

describe('OrganismsTemplateComponent', () => {
  test('renders as expected', () => {
    const tree = renderer.create(<OrganismsTemplateComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
