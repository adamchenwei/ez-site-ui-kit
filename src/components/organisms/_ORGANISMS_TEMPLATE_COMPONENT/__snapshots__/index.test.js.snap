// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrganismsTemplateComponent renders as expected 1`] = `
<h1
  className=""
  style={Object {}}
>
  Success 
  
</h1>
`;
